source/renderer/app/components/staking/delegation-center/WalletRow.js

```diff
@@ -364,38 +346,31 @@ export default class WalletRow extends Component<Props, WalletRowState> {
                 )}
               </div>
               <SVGInline svg={arrow} className={styles.arrow} />
+
               <div className={futureStakePoolTileStyles}>
                 {futurePendingDelegationStakePoolId ? (
-                  <div
-                    onMouseEnter={this.handleShowTooltip}
-                    onMouseLeave={this.handleHideTooltip}
-                  >
+                  <>
                     {futurePendingDelegationStakePool ? (
-                      <PopOver
-                        key="stakePoolTooltip"
-                        placement="auto"
-                        maxWidth={280}
-                        appendTo={'parent'}
-                        isShowingOnHover={false}
-                        isVisible={highlightedPoolId}
-                        themeVariables={popOverThemeVariables}
-                        allowHTML
-                        content={
-                          <TooltipPool
-                            stakePool={futurePendingDelegationStakePool}
-                            isVisible
-                            currentTheme={currentTheme}
-                            onClick={this.handleHideTooltip}
-                            onOpenExternalLink={onOpenExternalLink}
-                            top={top}
-                            left={left}
-                            color={stakePoolRankingColor}
-                            showWithSelectButton={showWithSelectButton}
-                            containerClassName={containerClassName}
-                            numberOfRankedStakePools={numberOfRankedStakePools}
-                            isDelegationView
-                          />
+                      <PoolPopOver
+                        openOnHover
+                        color={stakePoolRankingColor}
+                        currentTheme={currentTheme}
+                        onClose={() =>
+                          this.setState({
+                            highlightedPoolId: false,
+                          })
+                        }
+                        onOpen={() =>
+                          this.setState({
+                            highlightedPoolId: true,
+                          })
```

Comment:
@DominikGuzei Can you isolate this to functions like `handleOpen` and `handleClose`. Not necessary but maybe more descriptive